src/navigation/HomeContainer/HomeContainer.tsx

```diff
@@ -70,19 +76,38 @@ function HomeContainer(): React.JSX.Element {
   }, []);
 
   useLayoutEffect(() => {
-    dispatch(hideSplashScreen(true)); // dismiss if not already dismissed when home screen shown.
+    dispatch(screensLoaded(true)); // dismiss if not already dismissed when home screen shown.
   });
 
-  return (
-    <Stack.Navigator>
-      <Stack.Screen
-        name="Bookings"
-        component={BookingScreen}
-        options={{
-          headerRight: logoutButton,
-        }}
-      />
-    </Stack.Navigator>
-  );
+  if (featureFlags?.tabBarEnabled) {
+    return (
+      <Tab.Navigator
```

Comment:
What happens if we navigate within a tab?

Reply:
We don't at the moment so there is no way for me to test anything here. I suspect tweaks will be needed once we do further development on this. I have tested the presentation sheets we use and alerts and they all work okay though